src/main/java/com/happeningnow/service/ServiceOrganizer.java

```diff
@@ -2,36 +2,16 @@
 
 import com.happeningnow.model.Organizer;
 import com.happeningnow.repository.OrganizerRepository;
-import lombok.AllArgsConstructor;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.PageRequest;
+import com.happeningnow.service.factory.ServiceFactory;
 import org.springframework.stereotype.Service;
-import java.util.Optional;
-import java.util.UUID;
 
 @Service
-public class ServiceOrganizer {
+public class ServiceOrganizer extends ServiceFactory<Organizer, OrganizerRepository> {
 
     private final OrganizerRepository organizerRepository;
```

Comment:
this variable is not being used, you may remove it.

Reply:
Ok